src/RocketLcpBeacon.js

```diff
@@ -282,69 +203,20 @@ class LcpBeacon {
             elementInfo.type === "bg-img-set" ||
             elementInfo.type === "picture";
 
-        return (isImageOrVideo || isBgImageOrPicture)
-            &&
+        return (isImageOrVideo || isBgImageOrPicture) &&
             this.performanceImages.some(item => item.src === elementInfo.src);
     }
 
-    _getFinalStatus() {
-        if ( '' !== this.errorCode ) {
-            return this.errorCode;
-        }
-
-        const scriptTime = ( new Date() - this.scriptTimer ) / 1000;
-        if ( 10 <= scriptTime ) {
-            return 'timeout';
+    _logMessage(msg) {
+        if (!this.config.debug) {
+            return;
```

Comment:
This function is already in `RocketBeacon.js`, so why not using it the one from there instead of creating the same function here.

Reply:
That's exactly as u did in the WPR branch :smile: 

https://raw.githubusercontent.com/wp-media/wp-rocket/8a8e80b51f09555d1e4e2f84fa90b64682c05309/assets/js/wpr-beacon.js

do u want me to change that?

Reply:
Umm I think yea, 
I think it's a mistake I've made to be honest, I haven't even noticed that's what I've done haha